infra/scripts/k8s-common-functions.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
can we combine this with infra/scripts/runner-helper.sh

Reply:
good point, done